tests/integrationv2/tox.ini

```diff
@@ -11,4 +11,8 @@ deps =
     pytest==5.3.5
     pytest-xdist
 commands =
-    pytest -rpfsq {env:TOX_TEST_NAME:""}
+    pytest --cache-clear -rpfsq \
+        --openssl-version={env:S2N_LIBCRYPTO:"openssl-1.1.1"} \
```

Comment:
Is "openssl-1.1.1" a default? Why is "openssl-1.1.1" specifically used here?

Reply:
We also need a comment here explaining why S2N_LIBCRYPTO can be used to determine the version of an openssl peer.

Reply:
This is how our build system works.